src/apply_conf.rs

```diff
@@ -335,4 +351,14 @@ mod tests {
         // cleanup
         fs::remove_dir_all(destination_dir)
     }
+
+    #[test]
+    fn generate_keyfile_destination_path() {
+        assert!(keyfile_destination_path("some-dir", "eth0")
+            .is_some_and(|p| p.into_os_string().eq("some-dir/eth0.nmconnection")));
+        assert!(keyfile_destination_path("some-dir", "eth0.1234")
+            .is_some_and(|p| p.into_os_string().eq("some-dir/eth0.1234.nmconnection")));
```

Comment:
For these `assert_eq!(keyfile_destination_path("some-dir", "eth0"), Some(Pathbuf::from("some-dir/eth0.nmconnection"))` would be better imho.

Reply:
Looks much simpler! Will address, thanks.